server/index.js

```diff
@@ -16,9 +16,9 @@ app.use(express.static(__dirname + '/../client/dist'));
 app.use('/products/:id', express.static(__dirname + '/../client/dist'));
 
 app.get('/product/:id', (req, res) => {
```

Comment:
Use an endpoint other than '/product(s)/:id'. Both Derrick and I used these endpoints and all will need to be different for proxy server. Consider /product/info/:id . I changed mine to /product/pricing/:id

Reply:
I'll change mine also.